README.md

```diff
@@ -305,7 +305,9 @@ Enter your vault address: 0x3320ad928c20187602a2b2c04eeaa813fa899468
 Enter comma separated list of API endpoints for execution nodes: https://example.com
 Enter comma separated list of API endpoints for consensus nodes: https://example.com
 Enter the network name: goerli
-Vault data and keystores have been successfully recovered at `~/.stakewise/{vault_addr}/keystores`.
+Found 24 validators, recovering...
+Generating keystores  [####################################]  100%
+Keystores for vault {vault} successfully recovered
```

Comment:
Having path in the message was great - user knows where to look for the keystores.